src/transformers/data/data_collator.py

```diff
@@ -883,6 +883,8 @@ def torch_call(self, examples: List[Union[List[int], Any, Dict[str, Any]]]) -> D
         return {"input_ids": inputs, "labels": labels}
 
     def tf_call(self, examples: List[Union[List[int], Any, Dict[str, Any]]]) -> Dict[str, Any]:
+        import tensorflow as tf
```

Comment:
Note for other reviewers: this import pattern is the one we are using in other functions in this file (we might want to reconsider it)